README.md

```diff
@@ -535,6 +581,43 @@ See also:
   * [v::min()](#vminmin)
   * [v::max()](#vmaxmax)
 
+#### v::bic(string $countryCode)
+
+Validates a BIC (Bank Identifier Code) for a given country.
+
+```php
+v::bic("de")->validate("VZVDDED1XXX"); //true
+v::bic("de")->validate("VZVDDED1"); //true
+```
+
+Theses country codes are supported:
+
+ * "de" (Germany)
+
+**Note:** It is not recommended to use the German validator with the default
+configuration. You should configure the BAV library with one of the following
+`DataBackendContainer` implementations: `PDODataBackendContainer` or 
+`DoctrineBackendContainer`. This is done preferably by creating the file
+*bav/configuration.php*:
+```php
+namespace malkusch\bav;
+
+$configuration = new DefaultConfiguration();
+
+$pdo = new \PDO("mysql:host=localhost;dbname=test");
+$configuration->setDataBackendContainer(new PDODataBackendContainer($pdo));
+
+return $configuration;
```

Comment:
I'm not sure we should keep it into our documentation, it looks more a recommendation for the library than of the rule in itself.

Any thoughts?